content/blog/2020-03-24-march-20-community-gems.md

```diff
@@ -78,7 +78,7 @@ dependencies (for example, a training datset tracked by DVC) have been found. So
 if you had a hypoethetical DVC pipeline whose final process was `deploy.dvc`,
 you could run `dvc repro -f deploy.dvc` to rerun the whole pipeline.
 
-### Q: What's the best way to orgnize DVC repositories if I have several training datasets shared by several projects? Some projects use only one dataset while other use several. [Can one project have `.dvc` files corresponding to different remotes?](https://discordapp.com/channels/485586884165107732/563406153334128681/670664813973864449)
+### Q: What's the best way to oragnize DVC repositories if I have several training datasets shared by several projects? Some projects use only one dataset while other use several. [Can one project have `.dvc` files corresponding to different remotes?](https://discordapp.com/channels/485586884165107732/563406153334128681/670664813973864449)
```

Comment:
Nice job spotting all these typos! Is this one supposed to be `organize`?

Reply:
oops 🙈 I will claim 

> Muphry's law is an adage that states: "If you write anything criticizing editing or proofreading, there will be a fault of some kind in what you have written."[1] The name is a deliberate misspelling of "Murphy's law". 

on this one!

Fixed in https://github.com/iterative/dvc.org/pull/2507/commits/fe96ab54eb59a73c426238f62c60fc04202b479e